Formula/bpytop.rb

```diff
@@ -25,8 +25,13 @@ class Bpytop < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec, "python3")
+    venv.pip_install_and_link resources
```

Comment:
Can we avoid having to link `psutil` by wrapping the executables in an env script?

Though I'm not sure why you need it linked if you're using the virtualenv Python to run things.

Reply:
It does not need to be linked and it works with just `venv.pip_install`.  I left this in there by mistake and will remove it.